api/server/models/email.js

```diff
@@ -13,7 +13,7 @@ module.exports = function (Email) {
         }
         Email.send({
             to: data.email,
-            from: 'manish@excellencetechnologies.in',
```

Comment:
https://docs.strongloop.com/display/public/LB/Environment-specific+configuration
all constants need to be defined here 
server/config.json.

even maindrill constants need to be defined in this file
so change where mandrill is also defined